### DISCUSSION TITLE
java-driver has recently been donated by Datastax to The Apache Software Foundation and the Apache Cassandra project

### DISCUSSION THREAD

**Discussion by stefanofornari** [ing-bank/cassandra-jdbc-wrapper#45] [General]
Hello,
I have just noticed, the driver has been donated to the Cassandra project. Any plan to update to latest apache artifacts?

**Comment by maximevw** [comment]
Hello @stefanofornari 

For now, the donation is in progress and there is no new version of the driver released by the ASF yet. The latest version is still 4.17.0.

You can track the donation status here: https://issues.apache.org/jira/plugins/servlet/mobile#issue/CASSANDRA-18611

Unfortunately, the process is not very fast and they did not migrate the issues backlog at the moment for example. So, it seems the driver development is frozen until the donation process is finished.

~Also, they don't plan to change the group id of the artifacts until the next major release (at least).~ You can check [CEP-8](https://cwiki.apache.org/confluence/plugins/servlet/mobile?contentId=225152485#CEP8:DataStaxDriversDonation-Compatibility,Deprecation,andMigrationPlan) for further information.
UPDATE: It seems they finally decided to change the group id: https://github.com/apache/cassandra-java-driver/commit/f11622308d031bf85047c4811e737aeb6ae236e9 😆 ... but it's not a problem :)

But be sure the driver will be updated if a new version becomes available.
